misc/history-2nd-merge-from-ellvrt.md

```diff
@@ -0,0 +1,143 @@
+## Context
```

Comment:
I think you should put that into the commit, can be just copied as it is to not make more work.

Reply:
I don't think it's a big deal, otherwise you can shorten it. Your choice.